docs/index.md

```diff
@@ -6,9 +6,12 @@ nav_order: 1
 
 [![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/google/osv.dev/badge)](https://api.securityscorecards.dev/projects/github.com/google/osv.dev)
 
-[osv.dev](https://osv.dev) is a [vulnerability database](https://osv.dev/list)
-and triage infrastructure for open source projects aimed at helping both open
-source maintainers and consumers of open source.
+OSV seeks to reduce the complexities of open-source software security
```

Comment:
per my comment on the doc, I wonder if it's better for us to start with this instead:

"OSV enables developers to identify known 3P OSS vulnerabilities that pose genuine risk to their application and its environment, so they can focus remediation efforts on the vulnerabilities that matter and sustainably manage vulnerabilities that do not affect them."

And then briefly expand on the schema, OSV.dev infra (this repo), and OSV-Scanner ?

Reply:
Adjusted in 7bbb953